RARFix/plugin.yml

```diff
@@ -1,7 +1,7 @@
 ---
 name: RARFix
 version: "1.007"
-api: 1.11.0
+api: 2.0.0
```

Comment:
This plugin is only useful for a certain PocketMine version. It is pointless after the RAR bug is fixed. (I made that bug, and I made a plugin to fix it, and I named the bug "RAR" "Revenge Against the Rich")